apps/gcsweb/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
With the Ingress API in GA, new fields have been introduced to manage the backends. https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource. 
Can be done as followup but would great to convert the ingresses in the PR. :-D

Reply:
Makes sense! :) 
I'm unfortunately not going to have time this week (heading out for PTO) but can do that next week if we're ok holding off until then.

Reply:
No rush about  this. Enjoy your PTO! :-)